olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -45,6 +45,13 @@ constexpr auto kLogTag = "DiskCache";
 constexpr auto kLevelDbLostFolder = "/lost";
 constexpr auto kMaxL0Files = 4;
 
+#if defined(_WIN32) && !defined(__MINGW32__)
+// FIXME: How this message looks on Windows?
```

Comment:
Why do we need this and even with ifdef

Reply:
I don't know how to check the error message on Windows since I couldn't find any examples of error messages but I assume it should be different from other platforms.
The issue here is that leveldb shows a `no space left` error as an `IOError`, and to get what actual IO error you need to parse the string representation of leveldb::Status.